packages/blockchain-wallet-v4-frontend/src/data/auth/sagas.ts

```diff
@@ -530,7 +525,7 @@ export default ({ api, coreSagas }) => {
         } catch (e) {
           // if it fails with user already being reset, shuold be allowed
           // to continue with flow
-          if (e.description === 'User reset in progress') {
+          if (e.status === 409) {
```

Comment:
would be cool to move 409 to be in const we can define more HTTP status codes and reuse when ever we need ;)

Reply:
ohh, good idea, thanks.